### ISSUE TITLE
scheduler.step called multiple times?

### ISSUE THREAD

**Issue by leoauri** [NoFish-528/encodec-pytorch#21]
Hi, I see `scheduler.step()` called in https://github.com/ZhikangNiu/encodec-pytorch/blob/main/train_multi_gpu.py#L65 (if `config.common.amp`) and in https://github.com/ZhikangNiu/encodec-pytorch/blob/main/train_multi_gpu.py#L99. Is this a bug?

**Comment by ZhikangNiu** [comment]
@leoauri amp training isn't work so don't use amp training

**Comment by leoauri** [comment]
Thanks for your prompt reply!
